<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#FFFFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="first">
      <item>
       <layout class="QGridLayout" name="parameters">
        <item row="0" column="2">
         <widget class="QLabel" name="server_parameters_const">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>                                                   server parameters</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QFormLayout" name="xmlParameters">
          <item row="0" column="0">
           <widget class="QLabel" name="format_const">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:#000000</string>
            </property>
            <property name="text">
             <string>format</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="format">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6;color:#000000;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="from_const">
            <property name="styleSheet">
             <string notr="true">color:#000000</string>
            </property>
            <property name="text">
             <string>from</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="from">
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6;color:#000000;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="to_const">
            <property name="styleSheet">
             <string notr="true">color:#000000</string>
            </property>
            <property name="text">
             <string>to</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="to">
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6;color:#000000;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="id_const">
            <property name="styleSheet">
             <string notr="true">color:#000000</string>
            </property>
            <property name="text">
             <string>id</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="id">
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6;color:#000000;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="color_const">
            <property name="styleSheet">
             <string notr="true">color:#000000</string>
            </property>
            <property name="text">
             <string>color</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="color">
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6;color:#000000;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="image_const">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>       received image</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="xml_parameters_const">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>Received xml parameters</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="image">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:#f6f6f6;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <layout class="QGridLayout" name="serverParameters">
          <item row="4" column="0">
           <widget class="QLabel" name="clientPort_const">
            <property name="styleSheet">
             <string notr="true">color:#000000</string>
            </property>
            <property name="text">
             <string>client port</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="serverMessageTime_const">
            <property name="text">
             <string>server message time</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="serverPort">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6 ;color:#000000;; border: none;border-bottom: 1px solid currentColor;</string>
            </property>
            <property name="text">
             <string>3333</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="changeServerPortButton">
            <property name="styleSheet">
             <string notr="true">color:#000000; border: none;color:#000000; border: none;border-bottom: 1px solid currentColor;</string>
            </property>
            <property name="text">
             <string>set</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="serverMessageTime">
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6;color:#000000;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="isServerRunning_const">
            <property name="styleSheet">
             <string notr="true">color:#000000</string>
            </property>
            <property name="text">
             <string>is connected</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="serverIp_const">
            <property name="styleSheet">
             <string notr="true">color:#000000</string>
            </property>
            <property name="text">
             <string>server ip</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="clientIp_const">
            <property name="styleSheet">
             <string notr="true">color:#000000</string>
            </property>
            <property name="text">
             <string>client ip</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="clientPort">
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6;color:#000000;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="serverIp">
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6 ;color:#000000;; border: none;border-bottom: 1px solid currentColor;</string>
            </property>
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <layout class="QHBoxLayout" name="offOnButtons">
            <item>
             <widget class="QPushButton" name="startConnectionButton">
              <property name="styleSheet">
               <string notr="true">color:#000000; border: none;border-bottom: 1px solid currentColor;</string>
              </property>
              <property name="text">
               <string>start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stopConnectionButton">
              <property name="styleSheet">
               <string notr="true">color:#000000; border: none;color:#000000; border: none;border-bottom: 1px solid currentColor;</string>
              </property>
              <property name="text">
               <string>stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="isConnectedToServer">
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6;color:#000000;</string>
            </property>
            <property name="text">
             <string>no</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="clientIp">
            <property name="styleSheet">
             <string notr="true">background:#f6f6f6;color:#000000;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="changeServerIpButton">
            <property name="styleSheet">
             <string notr="true">color:#000000; border: none;color:#000000; border: none;border-bottom: 1px solid currentColor;</string>
            </property>
            <property name="text">
             <string>set</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="serverPort_const">
            <property name="styleSheet">
             <string notr="true">color:#000000</string>
            </property>
            <property name="text">
             <string>server port</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="textAndButton">
        <item>
         <widget class="QPlainTextEdit" name="text">
          <property name="minimumSize">
           <size>
            <width>900</width>
            <height>300</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#000000; background:#f6f6f6;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="BottomButton">
          <item>
           <widget class="QPushButton" name="sendRequestToServerButton">
            <property name="styleSheet">
             <string notr="true">color:#000000; border: none; color:#000000; border: none;border-bottom: 1px solid currentColor;</string>
            </property>
            <property name="text">
             <string>request data</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
